catalog-rest-service/src/main/resources/json/schema/metadataIngestion/databaseServiceQueryLineagePipeline.json

```diff
@@ -0,0 +1,48 @@
+{
+  "$id": "https://open-metadata.org/schema/entity/services/ingestionPipelines/databaseServiceQueryLineagePipeline.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "DatabaseServiceQueryLineagePipeline",
+  "description": "DatabaseService Query Lineage Pipeline Configuration.",
+  "definitions": {
+    "databaseLineageConfigType": {
+      "description": "Database Source Config Usage Pipeline type",
+      "type": "string",
+      "enum": ["DatabaseLineage"],
+      "default": "DatabaseLineage"
+    }
+  },
```

Comment:
Should not be this a separate Enum like `LineageConfigType`?

Reply:
we are building this "hack" to force the datamodel codegen tool to create different types. This way just by parsing the `type` from JSON to pydantic, we can be sure we are already picking the right class. We can have a chat about it if you're interested, there might be a better solution, but that's the approach we are following everywhere